lib/tabular_form_builder.rb

```diff
@@ -128,7 +128,7 @@ def field_container_wrap_field(field_html, options = {})
     if options[:no_label]
       field_html
     else
-      content_tag(:span, field_html, class: 'form--field-container')
+      content_tag(:span, field_html, class: options[:no_class] ? '' : 'form--field-container')
```

Comment:
Line is too long. [94/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.